frontend/src/modules/SettingModule/GeneralSettingsModule/forms/GeneralSettingForm.jsx

```diff
@@ -22,8 +24,10 @@ export default function GeneralSettingForm() {
           },
         ]}
       >
+        <p style={{display:"none"}}>{langCode}</p>
```

Comment:
I think this line doesn't serve any purpose and we can just remove it since the display is already "none"

Reply:
Due to how Form.Item works in Ant Design when used with controlled components. When the value prop of the Select component inside Form.Item changes, it doesn't trigger a re-render of the Form.Item itself, that's why that line is needed to trigger the UI to re-render